app/scripts/pages/connections.js

```diff
@@ -1,8 +1,7 @@
-'use strict'
 var React = require('react')
-var ConnectionList = require('../views/connectionlist.jsx')
-var Globe = require('../views/globe.jsx')
-var getLocation = require('ipfs-geoip').lookupPretty
+var ConnectionList = require('../views/connectionlist')
```

Comment:
saving that for later, there is so much to do here that I'm trying to do one step at a time :)